### DISCUSSION TITLE
Is it viable to extend tailwindcss/forms in the next versions?

### DISCUSSION THREAD

**Discussion by brendomaciel** [saadeghi/daisyui#2232] [Ideas / feature requests]
_First of all, I would like to say that I appreciate all the time and effort invested on this library by all the contributors._

Currently I'm facing [this problem](https://github.com/saadeghi/daisyui/issues/1431).

Usually, the proposed solution involves using the `strategy: 'class'` option. But in my case, as the project I'm working on have a considerable size, it would require a lot of work to find and replace all the DaisyUI specific classes in order to add the new prefix.

So, I would like to know if it's viable to use `@tailwindcss/forms` under the hood to implement the DaisyUI form elements because:
1. Would solve this problem
2. The `@tailwindcss/forms` is an official Tailwind plugin, so it make sense to support it
3. As we can see [here](https://github.com/saadeghi/daisyui/issues/1689), there are multiple issues about that
4. As @saadeghi says in [this comment](https://github.com/saadeghi/daisyui/issues/1431#issuecomment-1347822839), they're both doing the same thing

Disclaimer: I'm proposing it as a completely stranger to the project, so I don't know nothing about the philosophy and/or the technical barriers.

**Comment by saadeghi** [comment]
It's not about "supporting" `@tailwindcss/forms`  
- `@tailwindcss/forms` applies styles to the form elements based on a design,
- `daisyUI` applies styles to the form elements (with class names) based on another design.

There's nothing to "support" or "not support".  
It's like mixing Material design and Bootstrap. They both do the same thing so there's no reason to use both in the same project. Likewise, it's pointless to use both `daisyUI` and `@tailwindcss/forms` on the same project.

Let me know if you have any questions.
